dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -1,137 +1,339 @@
+                  @updateFiscalYearEndHandler="updateFiscalYearEndHandler"
+                />
+
+                <div class="uploadFormSection">
+                  <div class="col-3 p-3 topicLabel">
+                    <h4 id="assurance" class="anchor title">Assurance</h4>
+                  </div>
+
+                  <!-- Level of assurance -->
+                  <div data-test="assuranceSection" class="col-9 formFields">
+                    <h3 class="mt-0">Assurance</h3>
+                    <FormKit name="assurance" type="group">
+                      <!-- Level of assurance -->
+                      <div class="form-field">
+                        <UploadFormHeader
+                          :name="euTaxonomyKpiNameMappings.assurance ?? ''"
+                          :explanation="euTaxonomyKpiInfoMappings.assurance ?? ''"
+                        />
+                        <div class="lg:col-4 md:col-6 col-12 p-0">
+                          <FormKit
+                            type="select"
+                            name="assurance"
+                            placeholder="Please chose..."
+                            :validation-label="euTaxonomyKpiNameMappings.assurance ?? ''"
+                            validation="required"
+                            :options="assuranceData"
+                          />
+                        </div>
+                      </div>
+                      <!-- Assurance provider -->
+                      <div class="form-field">
+                        <UploadFormHeader
+                          :name="euTaxonomyKpiNameMappings.provider ?? ''"
+                          :explanation="euTaxonomyKpiInfoMappings.provider ?? ''"
+                        />
+                        <FormKit
+                          type="text"
+                          name="provider"
+                          :placeholder="euTaxonomyKpiNameMappings.provider ?? ''"
+                          :validation-label="euTaxonomyKpiNameMappings.provider ?? ''"
+                        />
+                      </div>
+
+                      <!-- Data source -->
+                      <div class="form-field">
+                        <FormKit type="group" name="dataSource">
+                          <h4 class="mt-0">Data source</h4>
+                          <div class="next-to-each-other">
+                            <div class="flex-1">
+                              <UploadFormHeader
+                                :name="euTaxonomyKpiNameMappings.report ?? ''"
+                                :explanation="euTaxonomyKpiInfoMappings.report ?? ''"
+                              />
+                              <FormKit
+                                type="select"
+                                name="report"
+                                placeholder="Select a report"
+                                validation-label="Select a report"
```

Comment:
Should be "Selecting a report" so that the error message will be "Selecting a report is required"